### PR TITLE
Explorer docs

### PR DESCRIPTION
**Pull request by adnejacobsen**
Added the "Usage and examples" section for Explorer

https://fmu-sumo.readthedocs.io/en/latest/explorer.html# is currently building from this branch so the changes can be previewed here. Will change back to master when this is merged.

### CODE REVIEW

File: docs/explorer.rst [hunk 1]
```diff
@@ -12,4 +12,395 @@ Api Reference
...
+
+    from fmu.sumo.explorer import Explorer 
+
+    sumo = Explorer()
+
+
+Authentication
+^^^^^^^^^^^^^^^
+If you have not used the `Explorer` before and no access token is found in your system, a login form will open in your web browser.
+It is also possible to provide the `Explorer` with an exisiting token to use for authentication, in this case you will not be prompted to login.
```
Comment (equinor-ruaj) [inline docs/explorer.rst:30]: "exisiting"

File: docs/explorer.rst [hunk 2]
```diff
@@ -12,4 +12,395 @@ Api Reference
...
+        print(case.name)
+        print(case.status)
+
+We can use the filter method to filter on the following properties:
+- uuid
+- name
+- status
+- user
+- asset
+- field
```
Comment (equinor-ruaj) [inline docs/explorer.rst:89]: I guess this is meant to render as a list, but readthedocs doesn't want to?
image

File: docs/explorer.rst [hunk 3]
```diff
@@ -12,4 +12,395 @@ Api Reference
...
+
+    cases = sumo.cases
+    cases = cases.filter(
+        user="peesv",
+        status="official"
+    )
+
+    assets = cases.assets
+
+The `CaseCollection.assets` propert gives us a list of unique values for the asset property in our list of cases.
```
Comment (equinor-ruaj) [inline docs/explorer.rst:125]: "propert"

File: docs/explorer.rst [hunk 4]
```diff
@@ -12,4 +12,395 @@ Api Reference
...
+    cases = cases.filter(
+        asset=assets[0]
+    )
+
+We can retrieve list of unique values for the following properties:
+- names 
+- statuses
+- users 
+- assets 
+- fields
```
Comment (equinor-ruaj) [inline docs/explorer.rst:151]: "a list" ?
Also, same as previous listing: does not render as such 😢

File: docs/explorer.rst [hunk 5]
```diff
@@ -12,4 +12,395 @@ Api Reference
...
+
+    my_case = sumo.get_case_by_uuid("1234567")
+
+
+Browsing data in a case
+^^^^^^^^^^^^^^^^^^^^^^^
+The `Case` object has properties for accessig different data types:
+- surfaces
+- polygons
+- tables
```
Comment (equinor-ruaj) [inline docs/explorer.rst:169]: should be "listy" ?

Comment (equinor-ruaj) [reply docs/explorer.rst:169]: also: "accessig"

File: docs/explorer.rst [hunk 6]
```diff
@@ -12,4 +12,395 @@ Api Reference
...
+
+The `SurfaceCollection.filter` method takes the following parameters:
+- uuid
+- name 
+- tagname 
+- iteration 
+- realization 
+- aggregation
+- stage 
+- time
```
Comment (equinor-ruaj) [inline docs/explorer.rst:216]: should be "listy"?

File: docs/explorer.rst [hunk 7]
```diff
@@ -12,4 +12,395 @@ Api Reference
...
+- uuid
+- name 
+- tagname 
+- iteration 
+- realization 
+- aggregation
+- stage 
+- time 
+
+All paramters suport a single value, a list of values or a `boolean` value.
```
Comment (equinor-ruaj) [inline docs/explorer.rst:218]: "paramters suport"

File: docs/explorer.rst [hunk 8]
```diff
@@ -12,4 +12,395 @@ Api Reference
...
+
+We can get list of filter values for the following properties:
+- names
+- tagnames 
+- iterations 
+- realizations
+- aggregations 
+- stages 
+- timestamps
+- intervals
```
Comment (equinor-ruaj) [inline docs/explorer.rst:250]: Listy?

**Review by equinor-ruaj — APPROVED** [review]
caught some typos 🎣 
LGTM

**Comment by adnejacobsen** [comment]
> caught some typos 🎣 LGTM

Good catch! 🐟 🐟 🐟

**Pull request merged**